### DISCUSSION TITLE
Module Management new module

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#1286] [Q&A]
Is there some magic need to get a module listed in the Module Management section of Oqtane?

**Comment by hishamco** [comment]
In which page ans section exctly

**Reply by leigh-pointer** [reply]
the page and section that you can install modules.  
I see some modules are available for download. The modules are picked up (i guess) from Nuget.  I would like to get my module listed here.

**Reply by hishamco** [reply]
I think there's a dicussion there about the module registration and discovery, @sbwalker was planned to change the way it working right now to use a custom registration and discovery instead of NuGet

**Reply by leigh-pointer** [reply]
Thanks, Ok got it! 
I will wait for the correct implementation.

**Reply by sbwalker** [reply]
currently it is based on having the tags node in your nuspec file:

```
    <tags>oqtane module</tags>
```

The Module Creator includes this tag for you... and the Blog sample module and Bootswatch theme also includes it.

**Reply by leigh-pointer** [reply]
```
<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
  <metadata>
    <id>StudioElf.Announcements</id>
    <version>1.0.0</version>
    ....
    <releaseNotes>Initial Release</releaseNotes>
    <copyright>StudioElf</copyright>
    <tags>oqtane module</tags>
    <dependencies>
      <dependency id="Oqtane.Framework" version="2.0.2" />
    </dependencies>
  </metadata>
```

I have that in the package and uploaded to Nuget some 5 hours ago.  Nothing is appearing as of yet.  Is there some kind of delay?

**Reply by sbwalker** [reply]
I assume so

**Reply by leigh-pointer** [reply]
ok, I will just be patient !

**Comment by leigh-pointer** [comment]
Pfff so package was there but will not showing in list because it is already installed, which is correct.
